config/conversion_issues.json

```diff
@@ -146,6 +146,40 @@
         "a": "more info"
       }
     },
+    {
+      "key": "lessons_hyperlinks",
+      "active": true,
+      "tool": "sakai.lessonbuildertool",
+      "icon-class": "icon-edit",
+      "description": "Lessons content hyperlinks",
+      "found": "Some Vula related hyperlinks were found in some lessons pages. Please see the table below for more details.",
+      "not_found": "Good to go! No Vula lessons hyperlinks found.",
+      "args": [
+        "lessons_soup"
+      ],
+      "info": {
+        "url": "https://docs.google.com/document/d/1m3xSY5gHPZ-Pw2KQcf5vcg73kW7CyuqVFI4fCl7mvvk/edit#heading=h.kszd2wgf91v7",
+        "a": "more info"
+      },
+      "hasLessonsDetails": true
+    },
+    {
+      "key": "lessons_tools",
+      "active": true,
+      "tool": "sakai.lessonbuildertool",
+      "icon-class": "icon-edit",
+      "description": "Lessons external tool",
+      "found": "Some Vula related tools were found in the lessons page. Please see the table below for more details.",
+      "not_found": "Good to go! No Vula lessons tools found.",
+      "args": [
+        "lessons_soup"
+      ],
+      "info": {
+        "url": "https://docs.google.com/document/d/1m3xSY5gHPZ-Pw2KQcf5vcg73kW7CyuqVFI4fCl7mvvk/edit#heading=h.kszd2wgf91v7",
+        "a": "more info"
+      },
+      "hasLessonsDetails": true
```

Comment:
How about we leave this out and it has detail if it returns detail (so None, True, False or a json with detail). There are other checks that aren't Lessons where it may be beneficial to return detail.

Reply:
The thinking here was, we wouldn't want to check every issue, rather just check selected issues for details. Selected issues would be marked by the `hasLessonsDetails` flag.

Reply:
Updated, see ticket for details/visuals of the tables.